code/index.html

```diff
@@ -1,21 +1,44 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>somewhere between paper an skin</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
 
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
-  </head>
-  <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
+  <nav>
+    <div class="nav"></div>
+    <ul>
+      <li><a href="#">/art work</a></li>
+      <li><a href="#">/tattoo art</a></li>
+      <li><a href="#">/about</a></li>
+      <li><a href="https://instagram.com/jessy.vagabunda?utm_medium=copy_link">/insta</a></li>
+      </ul>
+   </nav>
 
-      <img src="example-recipe-background.png" alt="Banana Bread">
+<div class="wrapper">
+  <div><img src="./Images/IMG_3058.jpg" alt="hund"></div>
+  <div><img src="./Images/IMG_3059.jpg" alt="hund"></div>
+  <div><img src="./Images/IMG_3060.jpg" alt="hund"></div>
+</div>
+<div>
+<div class="grey-box">
+    <h2>somewhere between paper an skin</h2>
+    <style>
+      img {
+      max: fullscreen;
+      height: auto;
+      }
+    </style>
+</div>
```

Comment:
Mmhh, looks like the styling for this image is broken or something is missing because the editor doesn't acknowledge the property "fullscreen". Also: maybe it would be a good idea to place all of your styling in the css file? 🙂